src/model/JobState.model.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Enumify } from 'enumify'
+
+/**
+ * Cylc valid job states.
+ */
+class JobState extends Enumify {
+  static SUBMIT_FAILED = new JobState('submit_failed')
```

Comment:
```suggestion
  static SUBMIT_FAILED = new JobState('submit-failed')
```

(The correct state name, and the css?, has a dash not an underscore)

Reply:
Yup, nice catch. Fixed.